toolchain/sem_ir/typed_insts.h

```diff
@@ -558,6 +558,22 @@ struct ConvertToValueAction {
   TypeInstId target_type_inst_id;
 };
 
+// A type for C++ `void`. Should only be used for pointers (`void*`).
+struct CustomCppVoidType {
```

Comment:
I'm not sure what's "custom" here -- perhaps remove it?

For contrast, in "custom layout type", I believe "custom" refers to the way layout is determined externally ("customized").

```suggestion
struct CppVoidType {
```

Reply:
Done.